tests/manage/z_cluster/nodes/test_automated_recovery_from_failed_nodes_reactive_IPI.py

```diff
@@ -219,12 +245,23 @@ def finalizer():
                     f"Successfully terminated node : "
                     f"{self.osd_worker_node[0].name} instance"
                 )
-            else:
+            elif get_node_status(self.osd_worker_node[0]) == constants.NODE_NOT_READY:
```

Comment:
get_node_status can return `None` as defined in https://github.com/red-hat-storage/ocs-ci/pull/5284/files#diff-1bba9ad83e489f5734ed993cec2d17018880017b2c2718e45607dcff3009f588R1917

I suggest you add a check the retuned value of `get_node_status()`

Reply:
@ebenahar If we remove the `try` and `exception` statements as suggested in the comment below https://github.com/red-hat-storage/ocs-ci/pull/5284#discussion_r775810701, should we still need to check the return value of `get_node_status()`?

Reply:
I think it's better to check the returned value to be sure the status is returned correctly

Reply:
Fixed